server/db/models/Product.js

```diff
@@ -0,0 +1,71 @@
+const db = require("../db");
+const { STRING, INTEGER, DECIMAL, TEXT } = require("sequelize");
+
+///////////////// PRODUCT MODEL /////////////////
+const Product = db.define("product", {
+  name: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
+  // Note: the brand column will be used to pair Product with Brand via script/seed.js.
+  // john - I comment this out because this is automatically create by the association in the indexjs
+  brandName: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
```

Comment:
you dontneed a brandname column- just use the association
```js
const cdg = await Brand.create({ name: 'comme des garcon' });
const shirt = await Product.create({ name: 'some shirt', brandId: cdg.id });
```

Reply:
mark for discussion